Source/ResourceLib.NetStandard/PixelFormat.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Vestris.ResourceLib
+{
+    public enum PixelFormat
```

Comment:
I think it's dangerous not to have the same type. (`System.Drawing.PixelFormat` vs `Vestris.ResourceLib.PixelFormat`). If you do not use those bitmaps functions I thinkg it will be wiser to disable them in .NET Standard.

Reply:
And for next major version as it's a breaking change, use `Vestris.ResourceLib.PixelFormat` for all target frameworks.

Reply:
Which would you prefer, move the custom PixelFormat enum into the net20 assembly, or disable the APIs using `#if !NETSTANDARD`? I can't quite tell.

Reply:
Sorry I was not clear. For this Merge Request just disable the code which is not .NET standard compatible. Using a `Vestris.ResourceLib.PixelFormat` will be done... one day.